packages/forma-36-react-components/src/components/DateTime/DateTime/DateTime.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { DateTimeFormat, formatDateTime } from '../dateUtils';
+import { CoercibleDate } from '../types';
+
+export interface DateTimeProps {
+  /**
+   * The date to display as a JS Date, an ISO8601 Timestamp, or Unix Epoch Milliseconds
+   *
+   * @example '2020-08-13T01:23:45.000Z'
+   * @example new Date()
+   * @example Date.now()
+   */
+  date: CoercibleDate;
+  /**
+   * Which display format to use
+   *
+   * @default 'FULL'
+   **/
+  format?: DateTimeFormat;
+  className?: string;
+  testId?: string;
+}
+
+/**
+ * Formats a JS Date object to an absolute date and time inside of a <time /> tag
+ */
+export const DateTime: React.FC<DateTimeProps> = ({
```

Comment:
@mshaaban0 starting a threaded conversation here re:styles on these components (thanks weird Github PR commenting scheme)

As we discussed in Slack I feel like these two components shouldn't have any styles applied to them, instead receiving whatever styles parent components define. In a sense these are plumbing components that can have porcelain applied by whatever context they exist in (Card, etc). Thoughts?

Regardless I'd rather get the issue of style or no style sorted out before merging as this will get ugly if assumptions about style are made that then get violated

Reply:
> As we discussed in Slack I feel like these two components shouldn't have any styles applied to them, instead receiving whatever styles parent components define. In a sense these are plumbing components that can have porcelain applied by whatever context they exist in (Card, etc). Thoughts?

I would agree!